sentry/Cargo.toml

```diff
@@ -17,27 +17,61 @@ autoexamples = true
 all-features = true
 
 [features]
-default = ["with_client_implementation", "with_default_transport", "with_panic", "with_failure"]
+default = ["client", "backtrace", "contexts", "failure", "panic", "transport"]
```

Comment:
failure being default kind of surprises me a little.  But I don't mind really.

Reply:
It is currently the default. I don’t want to break that just now, rather wait another release cycle to change the defaults.